installer/helm/chart/volcano/templates/controllers.yaml

```diff
@@ -47,7 +47,7 @@ rules:
     verbs: ["get", "list", "watch", "create", "delete", "update"]
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["get", "list", "watch", "create", "delete", "update"]
```

Comment:
seems that we only uesed get, create, delete, update verbs currently, so list and watch should be both removed: )

Reply:
Dibe